packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+
+### Added
+
+- Add support for locales on push notifications ([#5392](https://github.com/MetaMask/core/pull/5392))
+
+### Changed
+
+- **BREAKING:** Bump `@metamask/keyring-controller` peer dependency to `^20.0.0` ([#5426](https://github.com/MetaMask/core/pull/5426))
+- **BREAKING:** Bump `@metamask/profile-sync-controller` peer dependency to `^9.0.0` ([#5426](https://github.com/MetaMask/core/pull/5426))
+- Rename notification exported `__fixtures__` folder to `mocks` ([#5379](https://github.com/MetaMask/core/pull/5379))
```

Comment:
Nit: this seems to be only test related ?
```suggestion

Reply:
Yes, these are test-related, but they are also included in the published version of the package. According to the README, they are used for Storybook development.

That said, we can probably remove this entry because at the end of the day the exports didn't change.